tests/test_cmfd_linear_source/test_cmfd_linear_source.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python
+
+import os
+import sys
+sys.path.insert(0, os.pardir)
+sys.path.insert(0, os.path.join(os.pardir, 'openmoc'))
+from testing_harness import TestHarness
+from input_set import PwrAssemblyInput
+import openmoc
+
+
+class CmfdPwrAssemblyTestHarness(TestHarness):
+    """An eigenvalue calculation for a 17x17 lattice with 7-group C5G7
+    cross section data."""
+
+    def __init__(self):
+        super(CmfdPwrAssemblyTestHarness, self).__init__()
+        self.input_set = PwrAssemblyInput()
+
+    def _create_geometry(self):
+        """Initialize CMFD and add it to the Geometry."""
+
+        super(CmfdPwrAssemblyTestHarness, self)._create_geometry()
```

Comment:
It is equivalent to 
        self.input_set.create_materials()
        self.input_set.create_geometry()

Reply:
This works as well, but I'd rather keep materials creation and geometry creation separate, it's a bit more clear that way